qsiprep/interfaces/images.py

```diff
@@ -623,3 +667,50 @@ def to_lps(input_img, new_axcodes=("L", "P", "S")):
         return reoriented_img
     else:
         return input_img
+
+class TSplitInputSpec(AFNICommandInputSpec):
+    in_file = File(
+        desc="input file to 3dTsplit4D",
+        argstr=" %s",
+        position=-1,
+        mandatory=True,
+        copyfile=False,
+    )
+    out_name = File(
+        mandatory=True,
+        desc="output image file name",
+        argstr="-prefix %s.nii",
+    )
+    digits = traits.Int(
+        argstr="-digits %d", desc="Number of digits to include in split file names"
+    )
+
+class TSplitOutputSpec(TraitedSpec):
+    out_files = OutputMultiPath(File(exists=True))
+
+class TSplit(AFNICommand):
+    """Converts a 3D + time dataset into multiple 3D volumes (one volume per file).
+    For complete details, see the `3dTsplit4D Documentation.
+    <https://afni.nimh.nih.gov/pub/dist/doc/program_help/3dTsplit4D.html>`_
+    """
+
+    _cmd = "3dTsplit4D"
+    input_spec = TSplitInputSpec
+    output_spec = TSplitOutputSpec
+
+    def _list_outputs(self):
+        """Create a Bunch which contains all possible files generated
+        by running the interface.  Some files are always generated, others
+        depending on which ``inputs`` options are set.
+        Returns
+        -------
+        outputs : Bunch object
+            Bunch object containing all possible files generated by
+            interface object.
+            If None, file was not generated
+            Else, contains path, filename of generated outputfile
+        """
+        outputs = self._outputs().get()
+        ext = '.nii'
+        outputs["out_files"] = sorted(glob.glob(os.path.join(os.getcwd(),'{outname}.**.nii'.format(outname=self.inputs.out_name))))
```

Comment:
Could you spread this across a couple lines? The linter on this repo got disabled somehow, but I think this might be too long of a line